apps/paf/translations/src/en/validation.json

```diff
@@ -175,7 +175,8 @@
   },
   "report-person-location-mobile": {
     "maxlength": "You can't use more than {{maxlength}} characters for your answer",
-    "notUrl": "Enter a valid answer"
+    "notUrl": "Enter a valid answer",
+    "internationalPhoneNumber": "Please enter a valid phone number"
```

Comment:
Remove the 'Please' and capitalise 'enter'

Reply:
Done